digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -158,7 +158,7 @@ private void runWithHeartbeat(TaskRequest request)
                                 ex.getError(cf).get());  // TODO is error set?
                     }
                 }
-                catch (RuntimeException ex) {
+                catch (RuntimeException | AssertionError ex) {
```

Comment:
minor: May I ask you to comment about the reason why we need to handle "AssertionError" here?

Reply:
In this case we can catch AssertionError which is thrown by operators.run() and around processing in runWithWorkspace(). For example EMR operator may throw AssertionError.

Reply:
If we don't catch AssertionError here, it will be caught by above modification, but I guess by catching ConfigEvalEngine related AssertionError in this place may make clear the cause. So I also catch it.